views/body-snippet.html

```diff
@@ -1,7 +1,9 @@
-{% if data.global.seoGoogleTagManager %}
+{% set seoOverRides = apos.settings.getOption('seoOverRides') %}
```

Comment:
This would need to be on a module we know will be there and that has an alias to reference this way. With that in mind I'd use the SEO module and assign it an alias such as `aposSeo`.

I'm not sure it needs the `seoOverRides` layer. You could simply add the option directly on the module. That also saves you having to write an additional conditional below to see if `seoOverRides` exists.